docs/backend/upgrading/version-specific-migration/upgrade-to-61.md

```diff
@@ -117,3 +117,85 @@ If you have an existing Plone 5.2 or 6.0 site and you migrate to 6.1, then migra
 -   If the `plone.app.discussion` Python package is _not_ in your setup, but the site has existing comments (discussions), then the migration code stops with an error.
     Apparently you _were_ using comments in your site.
     Add the `plone.app.discussion` package to your dependencies, and run the migration again.
+
+
+## Distributions
+
+Plone 6.1 introduces the concept of a Plone {term}`distribution`.
+A Plone distribution is a Python package that defines specific features, themes, add-ons, and configurations that get activated when creating a Plone site.
+Now it is available in core Plone as the recommended way for creating a Plone site.
+
+When you install the [`Plone` 6.1 Python package](https://pypi.org/project/Plone/), you get several new packages and two distributions:
+
+-   [`plone.distribution`](https://github.com/plone/plone.distribution) is the new main package that offers the basis for distributions.
+-   [`plone.exportimport`](https://github.com/plone/plone.exportimport) imports and exports content, users, and other objects between Plone sites.
+    `plone.distribution` uses it.
+-   [`plone.volto`](https://github.com/plone/plone.volto) is the distribution to create a Plone site with the default frontend Volto.
+    This package was already present in Plone 6.0, but was upgraded to a distribution.
+-   [`plone.classicui`](https://github.com/plone/plone.classicui) is a new package and is the distribution to create a Plone site with the Classic UI frontend.
+
+When you start Plone, the launch screen prompts you to choose a distribution when creating a new Plone site.
+
+````{card}
+```{image} /backend/upgrading/version-specific-migration/images/distribution-chooser.png
+:alt: Launch screen for choosing a distribution
+:target: /_images/distribution-chooser.png
+```
++++
+_Launch screen for choosing a distribution_
+````
+
+After you select a distribution, you will fill out a brief form to configure your new Plone 6.1 site, similar to the process for Plone 6.0.
```

Comment:
I'm reconsidering lines 137-148 for inclusion in the Upgrade Guide. I think they belong instead in one of the [Install](https://6.docs.plone.org/install/) guides, but I'm not sure which, if any of them. I think the UI is going to change with future 6.1.x, and we almost never update an Upgrade Guide after its release, so I don't want these details and screenshot to become stale.